### PR TITLE
Harmonize the size and alignment of the DAM filters

### PR DESCRIPTION
**Pull request by fichtnerma**
---
Unify DAM Filter Dropdowns

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: COM-1029
-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>

image

</details>

### CODE REVIEW

**Comment by thomasdax98** [comment]
@fichtnerma It would be great if it also aligned with the search field. Could you look into that too?

File: .changeset/green-ants-play.md
Comment (johnnyomair) [inline .changeset/green-ants-play.md:5]:
```suggestion
Harmonize the size and alignment of the DAM filters
```

**Review by johnnyomair — COMMENTED** [review]
Shouldn't this be a Data Grid toolbar?

**Comment by fichtnerma** [comment]
> Shouldn't this be a Data Grid toolbar?

Part of this ticket is only fixing the alignment of the filters. I think the switch to datagrid will be done seperatly

**Review by johnnyomair — APPROVED** [review]
LGTM. Please wait for @thomasdax98' approval

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**
